### PR TITLE
[Channel] Removed fluent interfaces

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php [hunk 1]
```diff
@@ -17,6 +17,9 @@
 use Sylius\Component\Core\Model\ChannelInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php]: Are you sure that Paweł is the author of this class? The only time one you add an `@author` note is when you add yourself :wink:

Comment (stloyd) [inline src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php]: In fact first author of this class was `Kristian Løvstrøm`.

File: src/Sylius/Component/Channel/spec/Sylius/Component/Channel/Model/ChannelSpec.php [hunk 2]
```diff
@@ -12,8 +12,10 @@
 namespace spec\Sylius\Component\Channel\Model;
 
 use PhpSpec\ObjectBehavior;
-use Sylius\Component\Channel\Model\ChannelInterface;
 
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
```
Comment (stloyd) [inline src/Sylius/Component/Channel/spec/Sylius/Component/Channel/Model/ChannelSpec.php]: IIRC some time ago there was decision to not put `@author` tags into `*Spec` files (skipping the fact that Paweł is not author of this spec).

@pjedrzejewski @aRn0D Right?

File: src/Sylius/Component/Channel/Context/ChannelContextInterface.php [hunk 3]
```diff
@@ -14,22 +14,16 @@
 use Sylius\Component\Channel\Model\ChannelInterface;
 
 /**
- * Provides the context of currently used channel.
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Channel/Context/ChannelContextInterface.php]: I suppose this comment can be useful :wink:

**Comment by michalmarcinkowski** [comment]
Thank you Janek! :+1:

**Pull request merged**